packages/inventory-compliance/src/Components/RulesTable/RuleDetailsRow.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Grid, GridItem, Stack, StackItem, Text, TextVariants } from '@patternfly/react-core';
+import ConditionalLink from '../../PresentationalComponents/ConditionalLink';
+
+const RuleDetailsRow = ({ rule }) => {
+    const { refId, description, identifier, references, rationale } = rule;
+    const key = `rule-child-row-${refId}`;
+
+    return <React.Fragment key={ key }>
+        <div className='margin-top-lg'>
+            <Stack id={ `rule-description-${key}` } className='margin-bottom-lg'>
+                <StackItem style={ { marginBottom: 'var(--pf-global--spacer--sm)' } }>
+                    <Text className='pf-c-form__label' component={ TextVariants.h5 }><b>Description</b></Text>
+                </StackItem>
+                <StackItem isFilled>{ description }</StackItem>
+            </Stack>
+            <Stack id={ `rule-identifiers-references-${key}` } className='margin-bottom-lg'>
+                <Grid>
+                    { identifier && identifier.length > 0 && <GridItem span={ 2 }>
+                        <Text className='pf-c-form__label' component={ TextVariants.h5 }><b>Identifier</b></Text>
+                        <Text>
+                            {
+                                identifier.map((ident, idx) => (
+                                    <ConditionalLink
+                                        href={ ident.system }
+                                        target='_blank'
```

Comment:
For the `_blank` target the `rel` has to be set to `noopener noreferrer` to prevent XSS. See [CWE-79](https://cwe.mitre.org/data/definitions/79.html).